src/borg/helpers/parseformat.py

```diff
@@ -738,7 +741,7 @@ def __init__(self, archive, format, *, json_lines=False):
             'ctime': partial(self.format_time, 'ctime'),
             'atime': partial(self.format_time, 'atime'),
         }
-        for hash_function in hashlib.algorithms_guaranteed:
+        for hash_function in self.__class__.hash_algorithms:
```

Comment:
self.hash_algorithms didn't work?

Reply:
That would work as well, going to update the PR.